src/simulator/object_registry.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+import object
+
+ObjectId = int
```

Comment:
Is there a reason you define `ObjectId` type twice (one here and one in `object.py`?

Reply:
No. I now use `ObjectId` in `ObjectRegistry` through `object.ObejctId`. Is that ok?